includes/wc-core-functions.php

```diff
@@ -185,7 +185,16 @@ function wc_get_template_part( $slug, $name = '' ) {
 			);
 		}
 
-		wp_cache_set( $cache_key, $template, 'woocommerce' );
+		// Remove ABSPATH from template cache to avoid issues with multi container having different paths.
+		if ( 0 === strpos( $template, ABSPATH ) ) {
```

Comment:
Perhaps using `WC_ABSPATH` could be considered to account for cases where WooCommerce is not running under `ABSPATH`, as in the context of our unit test. That should provide the same behaviour as previously (so hopefully not break bw compat) and shouldn't be a problem when `ABSPATH`==`WC_ABSPATH`?